apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -194,6 +197,16 @@ function ListBoxInline({ options, layout }) {
     setShowSearch(newValue);
   };
 
+  const getActionToolbarProps = (isPopover) =>
+    listboxGetActionProps({
+      isPopover,
+      showToolbar,
+      containerRef,
+      isLocked,
+      listboxSelectionToolbarItems,
+      selections,
+    });
+
```

Comment:
Move this function outside of the `ListBoxInline` component to prevent cluttering and easier optimisation… if relevant.

Reply:
It may even be worth considering wrapping the `<ActionsToolbar />` inside another component that hosts this logic 🤔

Reply:
Ok, can just move the func outside/to its own file or outside of the component (I recently cleaned another file which had a lot of inline funcs like these 😬).